src/devices/Pn532/samples/Program.cs

```diff
@@ -53,7 +54,7 @@
 }
 else if (choiceInterface is { KeyChar: '2' })
 {
-    pn532 = new Pn532(I2cDevice.Create(new I2cConnectionSettings(1, Pn532.I2cDefaultAddress)), debugLevel);
+    pn532 = new Pn532(I2cDevice.Create(new I2cConnectionSettings(2, Pn532.I2cDefaultAddress)), debugLevel);
```

Comment:
Why this change from 1 to 2?

Reply:
I've changed becuase I was connecting P532 to bus 2, will undo that change